frontend/app/model/Eventbrite.scala

```diff
@@ -220,25 +220,10 @@ object Eventbrite {
     val fewerMembersTicketsThanGeneralTickets = member.quantity_total < generalRelease.quantity_total
   }
 
-  case class EBEvent(
-    name: EBRichText,
-    description: Option[EBRichText],
-    url: String,
-    id: String,
-    start: DateTime,
-    end: DateTime,
-    created: Instant,
-    venue: EBVenue,
-    capacity: Int,
-    ticket_classes: Seq[EBTicketClass],
-    status: String
-  ) extends EBObject {
-
-    val times = EventTimes(created, start)
-    val startAndEnd = new Interval(start, end)
+  case class EventWithDescription(ebEvent: EBEvent, ebDescription: EBDescription) {
```

Comment:
EventWithDescription wraps the two new classes, and the methods are distributed as appropriate between the three classes that result, but not many changes